packages/cxl-ui/scss/cxl-course-card.scss

```diff
@@ -0,0 +1,245 @@
+
+  // @see https://github.com/conversionxl/aybolit/pull/293
+  --video-background: hsla(355.8, 74.7%, 48%, 0.03); // --lumo-primary-color-3pct does not exist
+  --training-background: hsla(0, 0%, 10%, 0.03); // --lumo-shade-3pct does not exist
+  --playbook-background: hsla(213, 100%, 62%, 0.03); // No similar base color exists
+
+  // Container / Media queries
+  @media #{mq.$small} {
+    .container > .attributes {
+      display: none;
+    }
+
+    header .info .attributes {
+      display: flex;
+    }
+  }
+
+  [empty] {
+    visibility: hidden;
+    user-select: none;
+  }
+}
+
+:host([hidden]) {
+  display: none;
+}
+
+:host(:first-child) {
+  margin-top: unset;
+}
+
+:host(:last-child) {
+  margin-bottom: unset;
+}
+
+:host([theme~="dark"]) {
+  background-color: var(--lumo-contrast);
+}
+
+:host([theme~="video"]) {
+  background-color: var(--video-background);
+}
+
+:host([theme~="training"]) {
+  background-color: var(--training-background);
+}
+
+:host([theme~="playbook"]) {
+  background-color: var(--playbook-background);
+}
+
+.container {
+  display: block;
+  gap: var(--lumo-space-s);
+  width: 100%;
+  transition: all 0.3s ease;
+
+  > .attributes {
+    padding-top: 0;
+  }
+
+  .content-wrapper, footer {
+    margin-top: var(--lumo-space-s);
+  }
+}
+
+.attributes {
+  display: flex;
+  align-items: flex-start;
+  align-self: stretch;
+  padding: var(--lumo-space-s) 0;
+  gap: var(--lumo-space-s);
+  color: var(--lumo-shade-60pct);
+}
+
+header {
+  display: flex;
+  align-items: start;
+  justify-content: space-between;
+  gap: var(--lumo-space-m);
+
+  .info {
+    display: flex;
+    flex: 1 0 0;
+    flex-direction: column;
+    align-items: flex-start;
+    gap: var(--lumo-space-xs);
+    align-self: stretch;
+    max-width: calc(100% - var(--lumo-space-m) - 80px);
+    overflow: hidden;
+
+    .title {
+      font-family: Roboto, sans-serif;
+      font-size: var(--lumo-font-size-xl);
+      font-style: normal;
+      font-weight: 700;
+      line-height: var(--lumo-line-height-xs);
+      color: #1A1A1A;
+    }
+
+    .attributes {
+      display: none;
+    }
+  }
+
+
+  .instructor-image {
+    width: 80px;
+    height: 92px;
+
+    img {
+      height: 80px;
+      overflow: hidden;
+      border-radius: 100px;
+    }
+  }
+
+  .tags .tag {
+    &:first-child, &.new {
+      color: var(--lumo-primary-color)
+    }
+
+    &:first-child {
+      text-transform: capitalize;
+    }
+  }
+}
+
+.tags {
+  display: flex;
+  gap: var(--lumo-space-s);
+  flex-wrap: wrap;
+  max-width: 100%;
+  min-height: 1.6em;
+  overflow: hidden;
+
+  ::slotted(span):not(:first-child) {
+    overflow: hidden;
+    text-overflow: ellipsis;
+  }
+}
+
+.content-wrapper {
+  min-height: calc(1.2em * 6); // Six standard line-heights
+  line-height: 20px;
+
+  .content {
+    min-height: calc(1.2em * 3 + 1.25em); // Four standard line-heights plus global inherited <p> block margins
+  }
```

Comment:
I see other component use `var(--lumo-line-height-*)` values. I experimented with `line-height` here and `--lumo-line-height-s` is the closest and I recommend changing values to lumo vars to keep consitency.

Reply:
Good catch, @pawelkmpt, I did not see the tags being cut on the second line there.  

About the 320px. example, it looks like the header tags are the main offender. Can I just prevent them from wrapping and hide the overflow with an ellipsis, and let the user hover to see a default tooltip (standard HTML title attribute) with the full list of tags? A little less padding under 360px might also help a lot

Reply:
> About the 320px. example, it looks like the header tags are the main offender. Can I just prevent them from wrapping and hide the overflow with an ellipsis, and let the user hover to see a default tooltip (standard HTML title attribute) with the full list of tags?

Sure thing